app/models/manageiq/providers/azure/cloud_manager/vm_or_template_shared/scanning.rb

```diff
@@ -1,4 +1,16 @@
 module ManageIQ::Providers::Azure::CloudManager::VmOrTemplateShared::Scanning
+  extend ActiveSupport::Concern
+
+  included do
+    supports :smartstate_analysis do
```

Comment:
This default logic should go into `app/models/vm_or_template/scanning.rb`
Then you can remove the method `validate_smartstate_analysis` too, because its the default then